RHDEVDOCS-3859 Review RNs for OCP 4.9.z bug fix update (#43463)
openshift/openshift-docs: release_notes/ocp-4-9-release-notes.adoc

* Previously, the log file contained information in a single line without any line breaks. In the current release, the log file contains the expected line breaks with additional line breaks around log headers. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1985080[*BZ#1985080*])

> * Previously, the browser used to crash or display a white blank page when you open the *Edit Deployment* page for a non-existing Deployment resource. In the current release, a 404 error page is rendered instead of a browser crash or a white blank page. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2002273[*BZ#2002273*])

Reviewer: ```suggestion
* Previously, opening the *Edit Deployment* page for a non-existent Deployment resource resulted in a blank browser tab. In the current release, opening the *Edit Deployment* page for a non-existent Deployment resource results in a 404 error page. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2002273[*BZ#2002273*])
```

Author: Done

[id="ocp-4-9-24-bug-fixes"]
==== Bug fixes

> * Previously, opening the *Edit Deployment* page for a non-existent Deployment resource resulted in a blank browser tab. In the current release, opening the *Edit Deployment* page for a non-existent Deployment resource results in a 404 error page. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2002273[*BZ#2002273*])

Reviewer 2: @rishumehra please check the instructions for usage of the term 'deployment' in the OCP docs GH Documentation guidelines.
